pdns/dnsdistdist/docs/reference/dq.rst

```diff
@@ -131,6 +131,14 @@ This state can be modified from the various hooks.
 
     :returns: A table of EDNSOptionView objects, indexed on the ECS Option code
 
+  .. method:: DNSQuestion::getElapsedUs -> double
+
+     .. versionadded:: 2.0.0
```

Comment:
Add 1.9.x version

Reply:
Changing it in rel/dnsdist-1.9.x will not help, but I need to do that in master indeed!

Reply:
Done in https://github.com/PowerDNS/pdns/pull/14947